src/unix/internal.h

```diff
@@ -145,7 +145,8 @@ enum {
 
 /* loop flags */
 enum {
-  UV_LOOP_BLOCK_SIGPROF = 1
+  UV_LOOP_BLOCK_SIGPROF = 1,
+  UV_LOOP_TRACE_NOTIFY = 2
```

Comment:
I can't see this flag being used in the code.

Reply:
look in unix/loop.c ... it's set as a loop->flags... purely informational right now.